functions/tsconfig.json

```diff
@@ -2,14 +2,18 @@
   "compilerOptions": {
     "module": "commonjs",
     "noImplicitReturns": true,
-    "noUnusedLocals": true,
+    "noImplicitAny": false,
+    "noUnusedLocals": false,
```

Comment:
This may be rules we want to ignore in the tests but not in the code base.

Reply:
If possible yes! Hoping in does not impact the tests otherwise you may have to create a `tsconfig.test.json` that overwrites these rules